model_utils/managers.py

```diff
@@ -96,18 +96,11 @@ def _chain(self, **kwargs):
         return super()._chain(**kwargs)
 
     def _clone(self, klass=None, setup=False, **kwargs):
-        if django.VERSION >= (2, 0):
```

Comment:
No longer need to import django at the top of this file?

Reply:
I removed the import